### PR TITLE
CDK credentials error test

### PR DESCRIPTION
**Pull request by odockal**
New cdk itests for checking erroneous state during cdk start up with wrong credentials passed into server adapter

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by odockal** [comment]
https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Devstudio/view/devstudio_integration_tests/job/cdk.itests/317/

**Comment by odockal** [comment]
Moving all label/shell/jobs names into one class in next PR.

File: test-framework/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/core/enums/CDKHypervisor.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	public static CDKHypervisor getDefaultHypervisor() {
+		switch(CDKRuntimeOS.get().getRuntimeName()) {
+			case "linux":
+				return KVM;
+			case "darwin":
+				return XHYVE;
+			case "win":
+				return HYPERV;
+			default:
+				return null;
```
Comment (jkopriva) [inline test-framework/org.jboss.tools.cdk.reddeer/src/org/jboss/tools/cdk/reddeer/core/enums/CDKHypervisor.java]: What happen, if we return null? Shouldn't we throw an exception?

**Comment by jkopriva** [comment]
+1

**Pull request merged**